### PR TITLE
[ARTS-477] - UI tests added around profile name

### PR DESCRIPTION
**Pull request by sameerapurini**
## Description
PR contains tests added around the staff name display on welcome page. Only one scenario is added to regression suite as part of the story 477.

### Changes
- ARTS-477

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const environment = {
   clientId: 'f352ce15-0142-4dfa-8e18-801ee6391557',
   trialName: 'mts-trial-ui',
   // either specify the gateway root or give the URLs  for the services individually
-  gatewayUrl: '',
+  gatewayUrl: 'https://as-kate477-sc-gateway-dev.azurewebsites.net/api', // TECH DEBT pending story ARTS-382 and connected work. At the moment (9/3/21) Sameera has no end to end test environment, so to enable her to develop her UI tests we are pointing at an azure deployed backend.  Temp fix pending following story.
```
Comment (KateSant) [inline src/environments/environment.ts:11]: @mdp0 Are you happy with this workaround going into main, pending the QA environment work / story 382 to provide the backend to frontend tests?  Did anything come out of your meeting yesterday afternoon on alternative solutions?

**Review by KateSant — COMMENTED** [review]
Are we all happy (@tamirkamara @mdp0) about the temporary workaround in environment.ts, pending the QA services environment work which is coming in story 382 ?  I have no other comments on this PR so I would approve .

File: src/environments/environment.ts (hunk 1 above)
Comment (mdp0) [reply src/environments/environment.ts:11]: This will be our approach for the time being - stick with this until we develop some much heavier automation.

**Review by mdp0 — APPROVED** [review]

**Review by tamirkamara — APPROVED** [review]

**Review by sameendph — APPROVED** [review]
approving

**Review by KateSant — APPROVED** [review]

**Pull request merged**